test/forkMainnet/GenericSwap.t.sol

```diff
@@ -50,6 +49,7 @@ contract GenericSwapTest is Test, Tokens, BalanceUtil, Permit2Helper, SigHelper
     uint24[] defaultV3Fees = [3000];
     bytes defaultTakerPermit;
     bytes alicePermit;
+    bytes swapData;
```

Comment:
nit: maybe rename it to `strategyData` to avoid confusion as the first argument of `executeSwap*` is also `swapData`.